grid/payments/create-product/maps/create-product.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jest/no-export */
+
+import { describe, expect } from '@jest/globals';
+import { SuperfaceTest } from '@superfaceai/testing';
+
+export async function createProduct(provider: string): Promise<string> {
```

Comment:
thought: I wonder whether it wouldn't be better to move this shared function somewhere higher in the hierarchy, e.g. `payments/test-utils.ts` or similar.
Also, I'd consider passing `testName` as a parameter, so the recording is tied to that particular test where the setup is done, instead of having one shared setup – but maybe that's not necessary.